test/__init__.py

```diff
@@ -16,8 +17,7 @@
 
 @runtime_checkable
 class IDictRepresentable(Protocol):
-    def as_dict(self) -> dict[str, Any]:
-        ...
+    def as_dict(self) -> dict[str, Any]: ...
```

Comment:
I guess this one basically **is** a type annotation stub… (and I couldn't think of a good docstring for it)